viewers/wicket/ui/src/main/java/org/apache/isis/viewer/wicket/ui/pages/jquery.isis.wicket.viewer.js

```diff
@@ -195,6 +195,37 @@ $(function() {
     });
 */
 
+	/*
+	 * Adapted from https://bootstrap-menu.com/detail-basic-hover.html
+	 * Changes:
+	 * - ignoring mouseleave events if hovering over a popover
+	 */
+	if (window.innerWidth > 992) {	
```

Comment:
This probably needs some more scrutiny, the event listeners should always be registered, but only react on the right screen size